lib/elixir/lib/system.ex

```diff
@@ -114,6 +114,11 @@ defmodule System do
     end
   end
 
+  # Returns OTP version that Elixir was compiled with.
+  defmacrop get_otp_version do
```

Comment:
I totally missed that! In that case maybe call it compiled_otp_release/0 to match endiness/0 and compiled_endiness/0?